core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -411,4 +414,11 @@ RestoreV4Response restoreDatabaseByName(@PathParam("workspaceId") Long workspace
     RestoreV4Response restoreDatabaseByNameInternal(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name,
             @QueryParam("backupLocation") String backupLocation, @QueryParam("backupId") String backupId,
             @QueryParam("initiatorUserCrn") String initiatorUserCrn);
+
+    @PUT
+    @Path("internal/{name}/rotate_certificates")
```

Comment:
internal endpoints should use crn instead of name

Reply:
all the other internal calls in StackV4Endpoint use the stack name in the path. I just follow the pattern.

Reply:
I talked with @horadla23 and it doesn't affect performance in this case, so I'm ok with the name